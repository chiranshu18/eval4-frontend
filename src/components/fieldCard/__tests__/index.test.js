import FieldCard from '../index';
import { render , screen} from '@testing-library/react';
import React from 'react';

describe('Field Card', () => {
  it('Should render correctly', () => {
    const {asFragment} = render(<FieldCard />);
    expect(asFragment()).toMatchSnapshot();
  });
});