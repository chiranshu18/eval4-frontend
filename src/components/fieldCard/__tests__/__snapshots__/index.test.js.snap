// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Field Card Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="field-card"
  >
    <div
      class="field-car-logo"
    >
      <h2>
        Ab
      </h2>
    </div>
    <div
      class="field-card-name"
    >
      <h2>
        NAME
      </h2>
    </div>
    <div
      class="field-card-data-type"
    >
      <h2>
        Text
      </h2>
    </div>
    <div
      class="edit-field"
    >
      <img
        alt=""
        src="icons2.png"
      />
    </div>
    <div
      class="delete-field"
    >
      <img
        alt=""
        src="icon2.png"
      />
    </div>
  </div>
</DocumentFragment>
`;
